cmd/minishift/cmd/config/config.go

```diff
@@ -100,6 +100,14 @@ var settings []Setting = []Setting{
 		callbacks:   []setFn{RequiresRestartMsg},
 	},
 	{
+		name: "hostfolders-automount",
```

Comment:
`minishift setup-users` is just a simplified version of `Add` that does not ask for the name `Users` and `uncpath`. It would still be mounted as `mountpath` + `Name` => `/mnt/sda1/Users`